src/components/tips/tip-card.tsx

```diff
@@ -55,26 +44,42 @@ const TipCard: React.FC<{tip: Tip; i: number}> = ({tip, i}) => {
             {tipCompleted && <span className="sr-only">(completed)</span>}
           </span>
           <div className="flex w-full items-center justify-center">
-            <Image
-              src={thumbnail}
-              alt=""
-              objectFit="cover"
-              layout="fill"
-              aria-hidden="true"
-              className="brightness-90 transition duration-300 group-hover:brightness-75 dark:brightness-50"
-            />
+            <Link
+              tabIndex={-1}
```

Comment:
tabIndex -1 has keyboard skip the image as a link and go straight to the next title link which is nice